### PR TITLE
Initial implementation of custom variable save

### PR DESCRIPTION
**Pull request by nabsul**

### CODE REVIEW

File: includes/class-wc-siftscience-admin.php [hunk 1]
```diff
@@ -506,5 +514,33 @@ private function get_reporting_checkbox_description() {
...
+			$nonce = sanitize_key( $_POST['nonce'] );
+			if ( ! wp_verify_nonce( $nonce ) ) {
+				return;
+			}
+
+			$data = $_POST;
+
+			if ( empty( $data ) ) {
+				return;
+			}
```
Comment (ramico) [inline includes/class-wc-siftscience-admin.php:535]: $data = $_POST; put this line first 
then compare $data rather than POST

**Review by ramico — CHANGES_REQUESTED** [review]

File: includes/class-wc-siftscience-admin.php [hunk 2]
```diff
@@ -506,5 +514,33 @@ private function get_reporting_checkbox_description() {
...
+
+		/**
+		 * Saves custom fields that aren't covered by WooCommerce's automatic method
+		 */
+		private function save_custom_fields() {
+			if ( ! isset( $_POST['nonce'] ) ) {
+				return;
+			}
+
+			$nonce = sanitize_key( $_POST['nonce'] );
```
Comment (ramico) [inline includes/class-wc-siftscience-admin.php:526]: we bind '_nonce'  do we have 'nonce' 
or is it regarding WOO....

Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:526]: I'm still testing that.

File: includes/class-wc-siftscience-admin.php [hunk 3]
```diff
@@ -506,5 +531,27 @@ private function get_reporting_checkbox_description() {
 			$message = 'Help us improve this plugin by automatically reporting errors and statistics. ';
 			return "$message More info <a target='_blank' href='$url'>here</a>.";
 		}
+
+		/**
+		 * Saves custom fields that aren't covered by WooCommerce's automatic method
+		 */
+		private function save_custom_fields() {
+			if ( ! isset( $_REQUEST['_woosift_nonce'] ) ) {
```
Comment (ramico) [inline includes/class-wc-siftscience-admin.php:539]: join with  ||

**Review by ramico — APPROVED** [review]

**Pull request merged**